index.php

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+      
+ <title><?php echo "Assignment 7"; ?></title>
+  <meta charset="utf-8"> 
+</head>
+<body>
+
+<?php 
+$meatloaf = 'Rice';
+echo $meatloaf;
+echo "<br>";
```

Comment:
ideally you would comment or delete out these extra/debugging outputs for final submission.